### PR TITLE
Improved project info; CompartmentReportMap improvements

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

**Comment by eile** [comment]
@tribal-tec : Use oss group as maintainer as in Bluebrain/Hello?

File: CMakeLists.txt [hunk 1]
```diff
@@ -17,7 +17,8 @@ set(VERSION_PATCH "0")
 set(VERSION_ABI 5)
 
 set(BRION_DESCRIPTION "The Blue Brain C++ I/O library")
-set(BRION_ISSUES_URL "https://github.com/BlueBrain/Brion/issues")
+set(BRION_MAINTAINER "Daniel Nachbaur <daniel.nachbaur@epfl.ch>")
```
Comment (tribal-tec) [inline CMakeLists.txt]: bbp-dev-vizcore please :)

Comment (tribal-tec) [inline CMakeLists.txt]: ^^ Or oss group, yes.

**Comment by tribal-tec** [comment]
+1 otherwise

File: CMakeLists.txt (hunk 1 above)
Comment (eile) [inline CMakeLists.txt]: done

**Pull request merged**